src/Workspaces/Core/Desktop/Workspace/Host/TemporaryStorage/TemporaryStorageServiceFactory.MemoryMappedInfo.cs

```diff
@@ -346,19 +319,6 @@ public IntPtr GetPointer()
                 {
                     return (IntPtr)_start;
                 }
-
-                /// <summary>
-                /// Acquire the fixed pointer to the start of the memory mapped view.
-                /// The pointer will be released during <see cref="Dispose(bool)"/>
-                /// </summary>
-                /// <returns>The pointer to the start of the memory mapped view. The pointer is valid, and remains fixed for the lifetime of this object.</returns>
-                private static byte* AcquirePointer(MemoryMappedViewAccessor accessor)
```

Comment:
I dont see this.

 ptr += accessor.PointerOffset;

before, it was needed to make pointer to point to right location.